packages/terra-tabs/src/Tabs.jsx

```diff
@@ -187,16 +188,22 @@ class Tabs extends React.Component {
       </CollapsedTabs>
     );
 
+    const getTabs = () => {
+      if (this.state.breakpoint === 'tiny') {
+        return collapsedTabs;
+      }
+      return collapsibleTabs;
+    };
     return (
       <ContentContainer
         {...customProps}
         className={tabsClassNames}
         fill={fill}
         header={(
-          <ResponsiveElement
-            tiny={collapsedTabs}
-            small={collapsibleTabs}
-          />
+          <ResponsiveElement onChange={value => (this.setState({ breakpoint: value }))}>
```

Comment:
Should we do something like the following so we only update state when transitioning between tiny and non-tiny?
```js
<ResponsiveElement
    onChange={value => {
        const showCollapsedTabs = value === 'tiny';

        if (this.state.showCollapsedTabs !== showCollapsedTabs) {
            this.setState({ showCollapsedTabs })
        }
    }}
>
```

Reply:
updated here https://github.com/cerner/terra-framework/pull/972/commits/b6df56caadf61ab4705b2bb550fb93eb061642b8.